dart/lib/src/protocol/sentry_event.dart

```diff
@@ -83,10 +85,9 @@ class SentryEvent {
   /// If this behavior is undesirable, consider using a custom formatted [message] instead.
   final dynamic throwable;
 
-  /// The stack trace corresponding to the thrown [exception].
-  ///
-  /// Can be `null`, a [String], or a [StackTrace].
-  final dynamic stackTrace;
+  /// The stack trace corresponding to the thrown [exception] or attachedStackTrace
+  /// see [SentryOptions.attachStackTrace]
```

Comment:
the docs are off, this is only used if the thrown `throwable` was null and `attachStackTrace` is enabled

Reply:
basically its only used for captureMessage() which doesnt have an exception to set the stackTrace

Reply:
the docs for `attachStackTrace` is pretty clear about that btw